File: Common/Models/Address.cs
using System.ComponentModel.DataAnnotations;

namespace Common.Models
{
    public record Address
    {
        [Required]
        public required string FirstName { get; set; }

        [Required]
        public required string LastName { get; set; }

        [Required]
        public required string Email { get; set; }

        [Required]
        public required string AddressLine { get; set; }

        [Required]
        public required string Country { get; set; }

        [Required]
        public required string ZipCode { get; set; }
    }
}

File: rorderapi/src/models/address.rs
use bson::doc;
use serde::{Serialize, Deserialize};

#[derive(Serialize, Deserialize, Clone, Debug)]
#[serde(rename_all = "PascalCase")]
pub struct Address {
    pub first_name: String,
    pub last_name: String,
    pub email: String,
    pub address_line: String,
    pub country: String,
    pub zip_code: String,
}